config_order.yaml

```diff
@@ -4,6 +4,13 @@ logging:
     level: "notice"
   sdcard:
     level: "notice" # changing this to debug slows down things too much and causes panic
+ip:
+  ethernet:
+    address: "192.168.3.20"
```

Comment:
[nitpicking] mixing double quotes and single quotes in the other robot's config file